build/redeemer/Interfaces.sol

```diff
@@ -53,4 +53,8 @@ interface IElementToken {
 
 interface IYieldToken {
 	function redeem(address from, address to, uint256 amount) external;
```

Comment:
apparently missed this.

interfaces should neer have arg names in them

sometimes that shit slips in... burn it. (some other ticket tho)

Reply:
yuk. there's a bunch in here.